lib/work_calendar/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module WorkCalendar
-  VERSION = '1.0.3'
+  VERSION = '1.0.4'
```

Comment:
@ksnitsky Have you manually changed gem version here? Any update on auto-increment?